src/main/proto/netflix/titus/titus_vpc_api.proto

```diff
@@ -13,10 +13,7 @@ enum Family {
 };
 
 message Address {
```

Comment:
Do we still need a Address message? Do we plan on adding anything else to it in the future?

Reply:
Yeah, in the future, looking at the Eureka model, they have stuff like public / private / local / global. I can see this used for allocating a public IP.